### DISCUSSION TITLE
When i use the private key with passphrase, i get the exception System.NotSupportedException:“Key 'ENCRYPTED' is not supported.”

### DISCUSSION THREAD

**Discussion by Alimjan124** [sshnet/SSH.NET#1361] [Q&A] [answered]
When i use the private key with passphrase, i get the exception System.NotSupportedException:“Key 'ENCRYPTED' is not supported.” 
this is my code :
`var client = new SshClient(sshHostName, sshPort, sshUserName, new PrivateKeyFile(sshKeyFilePath, sshKeyFilePassword))`

and my private key generated with `openssl genpkey -algorithm RSA -aes-256-cbc -outform PEM -out private_key.pem -pkeyopt rsa_keygen_bits:2048`

**Comment by Ben555555** [comment, marked as answer]
See https://github.com/sshnet/SSH.NET/issues/1535
